src/test/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactoryUnitTest.java

```diff
@@ -1268,6 +1348,15 @@ Object[][] provideDataForTestCalculateGcContent() {
     //==================================================================================================================
     // Tests:
 
+    @Test ( dataProvider = "provideTestForMANESelectMode" )
+    void testMANESelectTranscriptSelectionCriteria(final List<GencodeGtfTranscriptFeature> inputTranscripts, final List<GencodeGtfTranscriptFeature> expectedTranscripts) {
+        final List<GencodeGtfTranscriptFeature> selectedTranscript = GencodeFuncotationFactory.retreiveMANESelectModeTranscriptsCriteria(inputTranscripts);
+        Assert.assertEquals(selectedTranscript.size(), expectedTranscripts.size());
+        for (GencodeGtfTranscriptFeature expectedTranscript : expectedTranscripts) {
+            Assert.assertTrue(selectedTranscript.contains(expectedTranscript));
+        }
+    }
+
```

Comment:
```suggestion
        @Test ( dataProvider = "provideTestForMANESelectMode" )
    void testMANESelectTranscriptSelectionCriteria(final List<GencodeGtfTranscriptFeature> inputTranscripts, final List<GencodeGtfTranscriptFeature> expectedTranscripts) {
        final List<GencodeGtfTranscriptFeature> selectedTranscripts = GencodeFuncotationFactory.retreiveMANESelectModeTranscriptsCriteria(inputTranscripts);
        Assert.assertEquals(selectedTranscripts.size(), expectedTranscripts.size());
        for (int i = 0; i < selectedTranscripts.size(); i++) {
            Assert.assertEquals(selectedTranscripts.get(i), expectedTranscripts.get(i));
        }
    }

```

Reply:
I can make this change, however i don't think we necessarily have a reason to enforce the order of the transcripts since after this method we are throwing them into a map anyway...